src/Queries/QueryRunner/QueryRunner.ts

```diff
@@ -158,9 +188,22 @@ export class QueryRunner {
       queryBuilder.where(where);
     }
 
+    for (const [property, value] of Object.entries(data)) {
+      if (typeof value === 'object') {
+        const symbols = Object.getOwnPropertySymbols(value);
+        for (const { description } of symbols) {
+          const operator = description as (typeof updateOperators)[number];
+          if (operator && isUpdateOperator[operator]?.(value)) {
```

Comment:
I think I missed checking if `UpdateOp.remove` is `true`, I just check for its' existence